seed/data_importer/tasks.py

```diff
@@ -743,6 +697,13 @@ def _save_raw_data_chunk(chunk, file_pk, progress_key):
                 raw_property.data_state = DATA_STATE_IMPORT
                 raw_property.organization = import_file.import_record.super_organization
                 raw_property.save()
+
+                if source_filename is not None:
+                    raw_property_state_to_filename[str(raw_property.id)] = source_filename
+
+            import_file.raw_property_state_to_filename.update(raw_property_state_to_filename)
```

Comment:
The way this is updated, I would think this **_avoids**_ a race condition if we have multiple tasks. But I may either be testing this weird or the race condition does actually happen.

<img width="1440" alt="Screen Shot 2020-04-29 at 4 41 03 PM" src="https://user-images.githubusercontent.com/30608004/80653825-3e861e00-8a38-11ea-8ac9-ef17ad53094d.png">

I uploaded the test zip file twice. For the first time, I allowed the 2 raw records to be created in one batch and it looks fine. For the second, I made the batch size 1 and saw that updating the zip only returned one key, value pair in the `raw_property_state_to_filename` field.